internal/cli/dialog/proposals_show_lineage.go

```diff
@@ -0,0 +1,48 @@
+package dialog
+
+import (
+	"fmt"
+
+	"github.com/git-town/git-town/v22/internal/cli/dialog/dialogcomponents"
+	"github.com/git-town/git-town/v22/internal/cli/dialog/dialogcomponents/list"
+	"github.com/git-town/git-town/v22/internal/cli/dialog/dialogdomain"
+	"github.com/git-town/git-town/v22/internal/forge/forgedomain"
+	"github.com/git-town/git-town/v22/internal/messages"
+	. "github.com/git-town/git-town/v22/pkg/prelude"
+)
+
+const (
+	proposalsShowLineageTitle = `Proposals Show Lineage`
+	proposalsShowLineageHelp  = `
+	How should Git Town share stack lineage in proposals?
```

Comment:
Since there are so many options here, what do you think about linking to the web page for this (assuming we add more details to that page)?

```suggestion
	How should Git Town share stack lineage in proposals?
	
	See https://www.git-town.com/how-to/github-actions-breadcrumb.html for details.
```